libs/cozy/mapping.ts

```diff
@@ -369,6 +390,14 @@ const makeYearFilterFunction = (field: AutofillField) => {
 
 const getValueInField = (field: AutofillField, regex: string): any => {
   for (const paperAttribute of PaperAutoFillConstants.PaperAttributes) {
+    // Special case for demande-logement-social.gouv.fr
+    if (field.htmlID.includes("montantMoins1")) {
+      return 2023;
+    }
+    if (field.htmlID.includes("montantMoins2")) {
+      return 2022;
+    }
+
```

Comment:
@zatteo  Est-ce qu'on ne préfère pas quelque chose comme ça ?
```suggestion
    // Special case for demande-logement-social.gouv.fr
    if (field.htmlID.includes("montantMoins1")) {
      return new Date().getFullYear() - 1;
    }
    if (field.htmlID.includes("montantMoins2")) {
      return new Date().getFullYear() - 2;
    }
```

Reply:
Si sûrement je vais adapter.